sources/engine/Xenko.Games/GameBase.cs

```diff
@@ -274,9 +231,8 @@ public bool IsMouseVisible
         {
             get
             {
-                return isMouseVisible;
+                return Window == null ? isMouseVisible : Window.IsMouseVisible;
```

Comment:
```suggestion
                return Window?.IsMouseVisible ?? isMouseVisible;
```

Using the null-coalesce operator prevent race condition where `Window` could change from `null` to `not-null` (or opposite) during the evaluation. And I personally think it is clearer 😉

Reply:
Yep, thanks.